dcpquery/etl/__init__.py

```diff
@@ -36,13 +40,121 @@ def transform_bundle(bundle_uuid, bundle_version, bundle_path, bundle_manifest_p
 
 
 def load_bundle(bundle, extractor, transformer):
-    bundle_row = Bundle(uuid=bundle["uuid"], version=bundle["version"], manifest=bundle["manifest"])
     bf_links = []
-
+    bundle_row = Bundle(uuid=bundle["uuid"], version=bundle["version"], manifest=bundle["manifest"])
     for f in bundle["files"]:
         filename = f.pop("name")
+        if filename == "links.json":
```

Comment:
Is there a way to condition this on the file content type or contents (schema identifier) instead of filename?

Reply:
could potentially check that the schema_type = link_bundle but I'm not sure if that is more stable than filename